doc/source/whatsnew/v1.5.0.rst

```diff
@@ -245,6 +245,18 @@ and attributes without holding entire tree in memory (:issue:`45442`).
 .. _`lxml's iterparse`: https://lxml.de/3.2/parsing.html#iterparse-and-iterwalk
 .. _`etree's iterparse`: https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.iterparse
 
+.. _whatsnew_150.enhancements.Optional_dependency_management:
+
+Optional dependencies version management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Optional pandas dependencies can be managed as extras in a requirements/setup file, for example:
+
+.. code-block:: python
+
+  pandas[recommended, s3]>=1.5.0
+
+Available optional dependencies are ``[all, recommended, computation, s3, gcp, azure, excel, parquet, feather,hdf5, sql-postgressql, ssql-mysql, sql-other, html, plot, table]`` (:issue:`39164`)
```

Comment:
should this list match setup.cfg?

Reply:
Good catch, yes this should match the names of the extra_options `{name}= {some_lib}>1.0.0`